src/main/resources/public/js/wallets.js

```diff
@@ -139,7 +139,8 @@ function getProjectWallets() {
  * @param owner Owner of the repo/project.
  * @param name Name of the repo.
  * @param type Type of the wallet (fake or stripe).
- * @todo #195#60min On frontend, enable wallet activation. Right now `activateWallet()` function is not used.
+ * @todo #253:30min On frontend, enable wallet activation.
+ *  Right now `activateWallet()` function is not used ('activate' button doesn't work).
```

Comment:
@criske I don't think we need this TODO, wallet activation works fine since a month ago I think.

Reply:
@amihaiemil on my end is not working
![Self XDSD](https://user-images.githubusercontent.com/10284893/102694007-fe71e680-4226-11eb-9c53-dc3e2bb7ca93.gif)

Reply:
@criske the button is inactive because the Limit is 0. Try setting up a Limit, then it should work.

Reply:
@criske and also at least a Payment Method.

Reply:
thanks, it works now.